Code/RDGeneral/LocaleSwitcher.cpp

```diff
@@ -33,7 +33,6 @@
 // LocaleSwitcher Dependencies
 #include <clocale>
 #ifndef _WIN32
```

Comment:
`xlocale.h` does seem to still be required with clang, but `<string>` is probably not required anywhere. 
Please try changing this to:
```
#include <clocale>
#ifdef __clang__
#include <xlocale.h>
#else
#include <locale.h>
#endif
```